src/Bem2d.jl

```diff
@@ -26,17 +26,81 @@ function obsgrid(xmin, ymin, xmax, ymax, npts)
     return xobs[:], yobs[:]
 end
 
+export Element
+struct Element
+    x1::Float64
+    y1::Float64
+    x2::Float64
+    y2::Float64
+    a::Float64
+    b::Float64
+    normalstress::Float64
+    # The following are derived properties (could be dynamic)
+    angle::Float64
+    length::Float64
+    halflength::Float64
+    xcenter::Float64
+    ycenter::Float64
+    rotmat::Array{Float64,2}
```

Comment:
Now these `rotmat`s could easily become an `SMatrix{2,2,Float64}` — but since you've already manually unrolled these calculations I didn't bother doing this as I think it'd not affect performance greatly.  It'd only ("only") be a nice aesthetic/ergonomic improvement.